### PR TITLE
Update: vulkan.version & Add: android.software.opengles.deqp.level-2022-03-01.xml 

### PR DESCRIPTION
**Pull request by chengrui-liu**
1. update android.hardware.vulkan.version-1_3.xml and android.software.vulkan.deqp.level-2022-03-01.xml
Tracked-On: OAM-103987
Signed-off-by: chengru1 chengrui.liu@intel.com
2. add android.software.opengles.deqp.level-2022-03-01.xml
Tracked-On: OAM-103986
Signed-off-by: chengru1 chengrui.liu@intel.com

### CODE REVIEW

File: groups/graphics/auto/product.mk [hunk 1]
```diff
@@ -109,12 +112,13 @@ PRODUCT_COPY_FILES += \
...
 
 PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.vulkan.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
+    frameworks/native/data/etc/android.software.vulkan.deqp.level-2022-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
 
 PRODUCT_PACKAGES += \
     vulkan.$(TARGET_BOARD_PLATFORM) \
     vulkan.pastel
 {{/vulkan}}
+
```
Comment (renchenglei) [inline groups/graphics/auto/product.mk:124]: Can we remove this line?

Comment (chengrui-liu) [reply groups/graphics/auto/product.mk]: had been removed

**Comment by renchenglei** [comment]
CC @feijiang1 & @YuanjunHuang 
Plz help take a review of this changes, thanks a lot.

**Review by feijiang1 — APPROVED** [review]

**Review by feijiang1 — APPROVED** [review]

**Comment by feijiang1** [comment]
LGTM

**Review by YuanjunHuang — APPROVED** [review]
LGTM

**Comment by renchenglei** [comment]
@swaroopbalan 
Can you help take a review of this changes, thanks a lot.

**Review by swaroopbalan — APPROVED** [review]

**Comment by chengrui-liu** [comment]
@swaroopbalan
Commit format error while preintegration, so I modified commit info.
Can you help take a review of these,  thanks a lot.

**Review by swaroopbalan — APPROVED** [review]

**Review by renchenglei — APPROVED** [review]

**Review by swaroopbalan — APPROVED** [review]

**Pull request merged**